lua/wikis/commons/Condition.lua

```diff
@@ -51,7 +52,11 @@ function ConditionTree:toString()
 	return table.concat(Array.map(self._nodes,
 		function(node)
 			if Class.instanceOf(node, ConditionTree) then
-				return String.interpolate('(${node})', {node = node:toString()})
+				local nodeString = node:toString()
+				if Logic.isEmpty(nodeString) then return end
```

Comment:
I think having a check on num of subNodes is clearer

Reply:
doesn't work
will add a test case that will work with the suggestion in this PR but fails with only checking the node count